config/zones/GB.yaml

```diff
@@ -5,17 +5,66 @@ bounding_box:
   - - 2.1533203125
     - 61.0795442346
 capacity:
-  biomass: 4438
-  coal: 1491
-  gas: 37047
-  geothermal: 0
-  hydro: 2063
-  hydro storage: 5603
-  nuclear: 4950
-  oil: 0
-  solar: 14518
-  unknown: 4775
-  wind: 26945
+  biomass:
+    - datetime: '2023-12-01'
+      source: bmreports.com
+      value: 4438
+    - datetime: '2022-12-01'
+      source: bmreports.com
+      value: 4572
+  coal:
+    datetime: '2022-12-01'
+    source: bmreports.com
+    value: 1491
+  gas:
+    - datetime: '2023-12-01'
+      source: bmreports.com
+      value: 37047
+    - datetime: '2022-12-01'
+      source: bmreports.com
+      value: 38985
+  hydro:
+    - datetime: '2023-12-01'
+      source: bmreports.com
+      value: 2063
+    - datetime: '2022-12-01'
+      source: bmreports.com
+      value: 4190
+  hydro storage:
+    - datetime: '2023-12-01'
+      source: bmreports.com
+      value: 5603
+    - datetime: '2022-12-01'
+      source: bmreports.com
+      value: 1928
+  nuclear:
+    - datetime: '2023-12-01'
+      source: bmreports.com
+      value: 4950
+    - datetime: '2022-12-01'
+      source: bmreports.com
+      value: 9256
```

Comment:
What happened here? That must be like 6-8 reactors closing? 👀

Reply:
![image](https://github.com/electricitymaps/electricitymaps-contrib/assets/107848894/b13d8e09-35da-46d7-aaab-b0f2f041ee89)

Reply:
https://assets.publishing.service.gov.uk/media/5a74eb21e5274a59fa715fc7/Table_of_past_and_present_UK_nuclear_reactors.pdf